### PR TITLE
me/summary: show trial information

### PR DESCRIPTION
**Pull request by goneri**
Jira Issue: <https://issues.redhat.com/browse/AAP-27332>

### Steps to test
1. Pull down the PR
2. ...
3. ...

## Production deployment

- [x] This code change is ready for production on its own
- [ ] This code change requires the following considerations before going to production:

### CODE REVIEW

File: ansible_ai_connect/main/tests/test_views.py [hunk 1]
```diff
@@ -185,3 +187,23 @@ def test_get_view(self):
...
+
+        r = self.client.get(reverse("me_summary"))
+        self.assertEqual(r.status_code, 200)
+        content = r.json()["content"]
+        expectation = """
+        Logged in as: test_user_name
+
+        User Type: Trial
+        Plan: trial of 90 days
+        Expiration: 2024-10-14
```
Comment (manstis) [inline ansible_ai_connect/main/tests/test_views.py:207]: This is likely to fail tomorrow or next week or next month?

**Review by manstis — COMMENTED** [review]
Am I missing something?

File: ansible_ai_connect/main/tests/test_views.py (hunk 1 above)
Comment (goneri) [reply ansible_ai_connect/main/tests/test_views.py:207]: ahah, this is embarrassingly true :-).

**Review by SiddharthRajaraman — APPROVED** [review]

**Review by SiddharthRajaraman — APPROVED** [review]

**Review by manstis — APPROVED** [review]
LGTM 👍 The future is safe in your hands.

**Pull request merged**